fast/stages/00-bootstrap/automation.tf

```diff
@@ -36,7 +36,8 @@ module "automation-project" {
   # machine (service accounts) IAM bindings
   iam = {
     "roles/owner" = [
-      module.automation-tf-bootstrap-sa.iam_email
+      module.automation-tf-bootstrap-sa.iam_email,
+      module.automation-tf-resman-sa.iam_email
```

Comment:
Why is this needed?

Reply:
Can you try with ` roles/cloudbuild.builds.editor` instead of owner for stage 1 SA?

Reply:
Yes, can be this 2 roles: _roles/source.admin, roles/cloudbuild.builds.editor_, but I faced a permission problem: 
_Error 403: generic::permission_denied: user does not have impersonation permission on the trigger service account specified: projects/mm-prod-iac-core-0/serviceAccounts/mm-prod-resman-net-1@mm-prod-iac-core-0.iam.gserviceaccount.com_, 
I tried to impresionate but not sure which SA should be implesionated to generate the Cloud Build trigger, I fixed using the owner role for the resman SA

Reply:
that is weird, to impersonate you need token creator role on the destination SA, which should be there; let me cross-check as soon as I have 10m

Reply:
I removed owner and used roles for cloud build and source repo, but we still have one issue which I'll details below in a separate comment